spogrebnyak/css_02/styles/lists.css

```diff
@@ -0,0 +1,26 @@
+pre {
+	margin-bottom: 0px;
+	margin-top: 0px;
+}
+.code{
+	background-color: black;
+	color: green;
+	width: 400px;
+	height: 4em;
```

Comment:
There is no requirements "height" in task description

Reply:
We have to use "max-height" property insted of "height". I've mistaked